src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -1,19 +1,21 @@
 @doc raw"""
-    CompressibleNavierStokesDiffusion2D(gamma, Re, Pr, Ma_inf, equations,
+    CompressibleNavierStokesDiffusion2D(gamma, mu, Pr, equations,
                                         gradient_variables=GradientVariablesPrimitive())
 
 These equations contain the diffusion (i.e. parabolic) terms applied
 to mass, momenta, and total energy together with the advective terms from
 the [`CompressibleEulerEquations2D`](@ref).
 
 - `gamma`: adiabatic constant,
-- `Re`: Reynolds number,
+- `mu`: dynamic viscosity,
 - `Pr`: Prandtl number,
-- `Ma_inf`: free-stream Mach number
 - `equations`: instance of the [`CompressibleEulerEquations2D`](@ref)
 - `gradient_variables`: which variables the gradients are taken with respect to.
                         Defaults to `GradientVariablesPrimitive()`.
 
+Fluid properties such as the dynamic viscosity mu can be provided in any unit system, e.g.,
```

Comment:
```suggestion
Fluid properties such as the dynamic viscosity ``\\mu`` can be provided in any consistent unit system, e.g.,
```
Should render this as a Greek letter